nilearn/_utils/niimg.py

```diff
@@ -44,14 +44,14 @@ def load_img(img, dtype=None):
     img: image
```

Comment:
Sorry I misread the diff completely but then a few liines above "or filepath" is not needed, right ?